components/dropdown/dropdown-content-mixin.js

```diff
@@ -438,7 +470,8 @@ export const DropdownContentMixin = superclass => class extends RtlMixin(supercl
 				this._position = position;
 			}
 
-			const maxHeight = Math.floor(this.openedAbove ? spaceAround.above : spaceAround.below);
+			const headerFooterHeight = headerRect.height + footerRect.height;
```

Comment:
Looks like the header + footer height is used above to compute `spaceRequired` -- could the declaration of this variable be moved up so both places can use it?